### DISCUSSION TITLE
List of Nixvim available colorschemes

### DISCUSSION THREAD

**Discussion by dedguy21** [nix-community/nixvim#1417] [Q&A] [answered]
Hi  I have used Google search to find out if there exists a list of the available color schemes for Nixvim, just to know which of them I might use before trying to download and custom install anything else.

Does such a list exist?

Thank you

**Comment by dedguy21** [comment, marked as answer]
Nevermind I just found it.

**Comment by andreifloroiu** [comment]
Well, Google brought me here :/

**Reply by traxys** [reply]
You can look in our documentation, they are listed in the sidebar (https://nix-community.github.io/nixvim/index.html)

**Comment by Dmutro4347** [comment]
Hi can i change themes on fly on nixvim like using themary

**Reply by MattSturgeon** [reply]
Nixvim works at build time, so it doesn't do anything on the fly at runtime.

However there's nothing stopping you from enabling multiple colourschemes in your nixvim config, then switching between them using the `:colorscheme` command.

Most of nixvim's colorscheme modules will add an extra line of lua code to enable itself, so the one you end up using by default will be whichever one ends up last in your `init.lua`. You could set this explicitly, late in `extraConfigLuaPre` or early in `extraConfigLua` (you can use `lib.mkBefore` and `lib.mkAfter` to influence an option's merge order).
